views/note.ejs

```diff
@@ -0,0 +1,26 @@
+<%- include("partials/head.ejs") %> <%- include("partials/header.ejs") %>
+
+<h1><%= job ? "Edit Note Entry" : "Add New Note Entry" %></h1>
+<form method="POST" action="<%= note ? '/notes/update/' + note.id : '/notes' %>" >
+  <input type="hidden" name="_csrf" value="<%= _csrf %>" />
+  <div>
+    <label for="title">Title:</label>
+    <input name="title" id="title" value="<%= note ? note.title : '' %>" />
+  </div>
+  <div>
+    <label for="subject">Subject:</label>
+    <input name="subject" id="subject" value="<%= note ? note.position : '' %>"/>
+  </div>
+  <div>
+    <label for="content">Content:</label>
+    <input name="content" id="content" value="<%= note ? note.content : '' %>"/>
+  </div>
+  <div>
+    <button onClick="submit"><%= note ? "Edit Note" : "Add New Note" %></button>
+    <a href="/notes">
+      <button type="button">Cancel</button>
+    </a>
+  </div>
+  <input type="hidden" id="note" name="note" value="<%= note ? note._id : '' %>" />
```

Comment:
You can remove this line, it's not necessary.

Reply:
Done